### PR TITLE
Michael x

### PR DESCRIPTION
**Pull request by MXSH-Dev**
- update file upload to include token in headers
- get user_id from redux and added auth token to request headers
- finished admin login functionalities
- fix image size issue

Next steps:
1. Add token to headers on all user related calls on frontend (did for get user info, add review, upload file, pending delete review)
2. fix key warning on course / instructor detail
3. finish add review modal

### CODE REVIEW

File: server/server/app.py [hunk 1]
```diff
@@ -481,6 +484,8 @@ def admin_login():
     email = request_data["email"]
     password = request_data["password"]
 
+    print(email, password)
```
Comment (aelgohar) [inline server/server/app.py:487]: ?

File: server/server/app.py [hunk 2]
```diff
@@ -470,6 +472,7 @@ def admin_signup():
 
 @app.route("/api/admin/login", methods=["POST"])
 def admin_login():
+    print('request received')
```
Comment (aelgohar) [inline server/server/app.py:475]: here too

File: server/server/app.py (hunk 1 above)
Comment (MXSH-Dev) [reply server/server/app.py:487]: for debug, forgot to remove

File: server/server/app.py (hunk 2 above)
Comment (MXSH-Dev) [reply server/server/app.py:475]: for debug, forgot to remove

File: client/src/pages/ExplorePage.tsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ export const ExplorePage: React.FC = () => {
               <h4
                 className={key === "professors" ? "text-white" : "text-primary"}
```
Comment (raysarinas) [inline client/src/pages/ExplorePage.tsx:25]: prob should change this to instructors as well

**Review by raysarinas — APPROVED** [review]
lgtm, just one small nitpick/comment

**Review by aelgohar — APPROVED** [review]

File: client/src/pages/ExplorePage.tsx (hunk 3 above)
Comment (MXSH-Dev) [reply client/src/pages/ExplorePage.tsx:25]: right

File: server/server/app.py (hunk 1 above)
Comment (MXSH-Dev) [reply server/server/app.py]: removed

File: server/server/app.py (hunk 2 above)
Comment (MXSH-Dev) [reply server/server/app.py]: removed

File: client/src/pages/ExplorePage.tsx (hunk 3 above)
Comment (MXSH-Dev) [reply client/src/pages/ExplorePage.tsx]: changed

**Pull request merged**